README.md

```diff
@@ -1 +1,68 @@
-# BookStore
\ No newline at end of file
+# BookStore
+The Bookstore is a website that display a list of books, add a book, and remove a selected book. This will explore the concepts of state management using Redux store, actions and reducers in React. Built with ReactJS.
+
+This project is part of the Microverse curriculum module 3. The main objective is to understand how create a React app, use components, hooks, state, run tests, and creating professional documentation for development.
+
+## Built With
+
+- Html/Css
+- Javascript ES6
+- React Js
+- Redux
+
+
+## Additional Tools
+
+- Webhint
+- Stylelint
+- Eslint
+- Lighthouse
+- Webpack
+
+## Getting Started
+
+To get a local copy of this project:
+
+Clone this repository or download the Zip folder:
+```
+https://github.com/VSeray/BookStore.git
+```
+ 
+In the project directory, you can run:
+
+### `npm start`
+
+### `npm test`
+
+### `npm run build`
+
+### `npm run eject`
+
+### `npm run build` fails to minify
```

Comment:
- [ ] Let's try to be detailed in our documentation as it can be of great help to users. Before users can run any of these commands, they need to install packages with `npm install`. Kindly include this command.
- [ ] **[OPTIONAL]** It would be nice to list (by the side) what these commands actually do.